cdxev/__main__.py

```diff
@@ -491,6 +503,81 @@ def create_merge_vex_parser(
     return parser
 
 
+# noinspection PyUnresolvedReferences,PyProtectedMember
+def create_vex_parser(
+    subparser: argparse._SubParsersAction,
+) -> argparse.ArgumentParser:
+    parser = subparser.add_parser(
+        "vex",
+        help="Executes commands [list, search, trim, extract] on VEX/SBOM files.",
```

Comment:
Should not include the commands itself to the help. Otherwise you always have to adjust and keep it in mind. The help-function itself will already show the subcommands.